pkg/render/manager.go

```diff
@@ -549,6 +551,11 @@ func (c *managerComponent) voltronContainer() corev1.Container {
 		mounts = append(mounts, c.cfg.TunnelServerCert.VolumeMount(c.SupportedOSType()))
 		mounts = append(mounts, c.cfg.VoltronLinseedKeyPair.VolumeMount(c.SupportedOSType()))
 	}
+	if c.cfg.NonClusterHost != nil {
+		// We need the internal tls key pair for voltron and linseed mtls connection
+		// when non-cluster host log ingestion is enabled.
+		mounts = append(mounts, c.cfg.InternalTLSKeyPair.VolumeMount(c.SupportedOSType()))
```

Comment:
This creates the possibility for the secret to be mounted twice. You should combine it with the if-statement just above this one.

Reply:
They are combined.